<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkerSelectClass</class>
 <widget class="QDialog" name="WorkerSelectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select worker</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbID">
       <property name="text">
        <string>Worker ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="leID">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>true</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbSection">
       <property name="text">
        <string>Worker Section</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leSection">
       <property name="maxLength">
        <number>64</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbName">
       <property name="text">
        <string>Worker Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leName">
       <property name="maxLength">
        <number>64</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbRemark">
       <property name="text">
        <string>Remark</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leRemark">
       <property name="maxLength">
        <number>64</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbNichiji">
       <property name="text">
        <string>Update Time</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="leUpdateTime">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>true</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="PasswordPushButton" name="pbOK">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>J:/検査画面icon/flash_swf_64.PNG</normaloff>J:/検査画面icon/flash_swf_64.PNG</iconset>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableWidget" name="twWorkerList"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PasswordPushButton</class>
   <extends>QPushButton</extends>
   <header>passwordpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
